src/constants/constants.js

```diff
@@ -10,13 +10,28 @@ export const CUSTOM_TYPES_BUTTONS = {
   standard: 'ui button'
 };
 
+export const DEFAULT_TYPES = {
+  standard: 'standard'
```

Comment:
Will there be more values here? It seems it should be a simple `const DEFAULT_CONTACT_TYPE = 'standard'`